src/components/Common/ProposalPreviewCard/CategoryModule.tsx

```diff
@@ -10,20 +10,19 @@ import './CategoryModule.css'
 import ProposalPreviewCardSection from './ProposalPreviewCardSection'
 
 interface Props {
-  proposal: ProposalAttributes
+  proposal: Pick<ProposalAttributes, 'type' | 'status' | 'configuration'>
   isHovered?: boolean
+  slim?: boolean
 }
 
-function CategoryModule({ proposal, isHovered }: Props) {
+function CategoryModule({ proposal, isHovered, slim = false }: Props) {
   const t = useFormatMessage()
   const isGrant = proposal.type === ProposalType.Grant
   const isBid = proposal.type === ProposalType.Bid
   return (
     <ProposalPreviewCardSection className="CategoryModule">
-      <div className="CategoryModule__Container">
-        <div className="CategoryModule__StatusPill">
-          <StatusPill status={proposal.status} />
-        </div>
+      <div className={classNames('CategoryModule__Container', slim && 'CategoryModule__Container--slim')}>
+        <div className="CategoryModule__StatusPill">{!slim && <StatusPill status={proposal.status} />}</div>
```

Comment:
Couldn't this be
```jsx
{!slim && (
  <div className="CategoryModule__StatusPill">
    <StatusPill status={proposal.status} />
  </div>
)}
```
or do you need that extra class?

Reply:
Maybe I don't need it.

I added the extra class so I could align the status pill with the ones in the proposals list.

<img width="827" alt="image" src="https://github.com/decentraland/governance/assets/2858950/a8a2eedc-e85a-4790-876e-c1c7c9776768">

So what I'm doing is not showing the pill but still occupying that space.